grafana-plugin/src/models/filters/filters.ts

```diff
@@ -35,6 +35,15 @@ export class FiltersStore extends BaseStore {
     }
   }
 
+  @action
+  set needToParseFilters(value: boolean) {
+    this._needToParseFilters = value;
+  }
+
+  get needToParseFilters() {
+    return this._needToParseFilters;
+  }
```

Comment:
Setters and getters have a different purpose in MobX. Getters are used to define `computeds` - they derive a value based on other observables. Setters are used for some really rare case, tbh I've never had such (to override computed mentioned before). More in docs: https://mobx.js.org/computeds.html#computed-setter 

Here we just need observable and action that can update it. Also this observable is a public property, I think underscore in the name is redundant. The name itself should also indicate that it is updating something, the verb should be used.

So to sum up I would suggest:
```
@observable
needToParseFilters = false;
...
 @action
 setNeedToParseFilters(value: boolean) {
    this.needToParseFilters = value;
  }

```

Reply:
fixed